src/autoupdate.c

```diff
@@ -29,55 +29,75 @@
 
 #include "swupd.h"
 
-static void print_help(const char *name)
+static void print_help()
 {
-	print("Usage:\n");
-	print("   swupd %s [options]\n\n", basename((char *)name));
 	print("Help Options:\n");
 	print("   -h, --help              Show help options\n");
 	print("       --enable            enable autoupdates\n");
 	print("       --disable           disable autoupdates\n");
 	print("\n");
+
+	global_print_help();
 }
 
-static int enable;
-static int disable;
+static bool enable = false;
+static bool disable = false;
 
 static const struct option prog_opts[] = {
 	{ "help", no_argument, 0, 'h' },
-	{ "enable", no_argument, &enable, 1 },
-	{ "disable", no_argument, &disable, 1 },
-	{ 0, 0, 0, 0 }
+	{ "enable", no_argument, 0, 'e' },
```

Comment:
We don't need short opts for enable and disable. Take a look at bundle-add for an example

Reply:
It is good to have short opts isnt it?

Reply:
We had several problems on conflicts of short opts before. As the number of short opts are limited (we have limited number of letters) we prefer to use it only when needed.

Reply:
ahh I see.